.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { config } = require('@dhis2/cli-style')
+
+module.exports = {
```

Comment:
I never added my `eslintrc`... This is what I ended up with

const { config } = require("@dhis2/cli-style");
//import { config } from "@dhis2/cli-style";

module.exports = {
    extends: [
        config.eslintReact,
        "plugin:@typescript-eslint/recommended",
        "plugin:import/typescript",
    ],
    parser: "@typescript-eslint/parser",
    plugins: ["@typescript-eslint"],
    root: true,
    rules: {
        "@typescript-eslint/no-var-requires": 0,
        "import/extensions": 0,
        "react/prop-types": 0, // we are using types instead
    },
};